compiler/passes/convert-typed-uast.cpp

```diff
@@ -5071,6 +5090,157 @@ bool TConverter::enter(const ExternBlock* node, RV& rv) {
 
 void TConverter::exit(const ExternBlock* node, RV& rv) {}
 
+static SymExpr* makeCaseCond(TConverter& tc, TConverter::RV& rv,
```

Comment:
IMO we should normalize either `TConverter*` or `TConverter&` for static helper functions so that they're consistent, I don't really have a preference here but usually just do `TConverter*` cause then you can pass in `this`. Thoughts?

Reply:
I guess I prefer to use references when possible, but for consistency I'll change this.

Reply:
I'm fine with using references, I'd just want to go and change the other areas to use them too 😄 is all. So feel free to keep and I'll add a TODO.